bolt/lib/Profile/StaleProfileMatching.cpp

```diff
@@ -307,25 +307,32 @@ void BinaryFunction::computeBlockHashes(HashFunction HashFunction) const {
 FlowFunction
 createFlowFunction(const BinaryFunction::BasicBlockOrderType &BlockOrder) {
   FlowFunction Func;
+  std::vector<uint64_t> ExitBlocksIndices;
```

Comment:
Again, we have to have a consistent definition of what is an exit block. It doesn't matter where you put it, another flag in `FlowBlock` or extra bookkeeping here. Why do we need to have extra bookkeeping given FlowBlock already has `isExitBlock`.